src/dcat-us/compile-dcat-feed.ts

```diff
@@ -25,6 +22,32 @@ function removeUninterpolatedDistributions(distributions: any[]) {
   return distributions.filter((distro) => !(typeof distro === 'string' && distro.match(/{{.+}}/)?.length));
 }
 
+function generateDcatItem(feedTemplate, feedTemplateTransforms, geojsonFeature) {
+  const defaultFields = {
+    '@type': 'dcat:Dataset'
+  };
+
+  // moving feature properties fields at the top level
+  // and geometry to second as the values for the properties
+  // can be easily referenced in template
+  const dcatFeedData = { 
+    ...geojsonFeature?.properties, 
+    ...{ geometry: geojsonFeature?.geometry } 
```

Comment:
I think this destructuring is unnecessary.  Can't you just do:

`geometry: geojsonFeature.geometry`

Reply:
Yes, we can remove the unnecessary destructuring.